app/presenters/fee/interim_fee_type_presenter.rb

```diff
@@ -8,18 +8,21 @@ def data_attributes
         legal_aid_transfer:   fee_type.is_retrial_new_solicitor?,
         trial_concluded:      fee_type.is_retrial_new_solicitor?,
         retrial_dates:        fee_type.is_retrial_start?,
-        ppe:                  unless_disbursement_or_warrant?,
-        fee_total:            unless_disbursement_or_warrant?,
+        ppe:                  is_not_disbursement_or_warrant?,
+        fee_total:            is_not_disbursement?,
         warrant:              fee_type.is_interim_warrant?,
         disbursements:        is_not_warrant?
     }
   end
 
-
   private
 
-  def unless_disbursement_or_warrant?
-    !(fee_type.is_disbursement? || fee_type.is_interim_warrant?)
+  def is_not_disbursement_or_warrant?
+    is_not_disbursement? && is_not_warrant?
```

Comment:
I think it's great to have renamed this method, but I think to make it even clearer what it does, it could be renamed is_neither_disbursement_nor_warrant